### PR TITLE
MON-3848: add e2e-aws-ovn-single-node techpreview jobs

### PR DESCRIPTION
**Pull request by machine424**

### CODE REVIEW

**Comment by machine424** [comment]
/payload-job-with-prs periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview openshift/cluster-monitoring-operator#2334 openshift/kubernetes-metrics-server#26 openshift/kubernetes#1960 openshift/machine-config-operator#4338

File: core-services/release-controller/_releases/priv/release-ocp-4.16.json [hunk 1]
```diff
@@ -138,6 +138,20 @@
                 "name": "periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-serial-priv"
             }
         },
+        "aws-ovn-single-node-techpreview": {
```
Comment (machine424) [inline core-services/release-controller/_releases/priv/release-ocp-4.16.json:141]: I had to add those `priv` entries, where are they used?

**Comment by machine424** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-single-node-techpreview periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

/retest

**Comment by machine424** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-single-node-techpreview periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

**Comment by machine424** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-single-node-techpreview periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

**Comment by slashpai** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

File: core-services/release-controller/_releases/priv/release-ocp-4.16.json (hunk 1 above)
Comment (eggfoobar) [reply core-services/release-controller/_releases/priv/release-ocp-4.16.json:141]: @bradmwilliams Would know for sure, but I think it had to do with CVE validation in the private environments by adding the `-priv` suffix.

Comment (bradmwilliams) [reply core-services/release-controller/_releases/priv/release-ocp-4.16.json:141]: You should only ever modify the files in: `core-services/release-controller/_releases` and then run a `make release-controllers` to automatically generate the files in: `core-services/release-controller/_releases/priv`.

Comment (machine424) [reply core-services/release-controller/_releases/priv/release-ocp-4.16.json:141]: Yes, I've run `make release-controllers` as requested [in the error message](https://github.com/openshift/release/blob/68d653bcd1431520f2121f4446ce81b6c3b7e056/hack/validate-release-jobs.py#L106C1-L106C121)
I just wanted to know more about those `priv` jobs.

**Comment by eggfoobar** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

Retrying job, that failure is odd and I'm investigating why that failed to install, but it seems like an install fluke.

**Comment by eggfoobar** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

**Comment by slashpai** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview

**Comment by eggfoobar** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview-serial periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-single-node-techpreview-serial

Fix for serial should be in latest nightly, trying

**Comment by eggfoobar** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-techpreview-serial periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-single-node-techpreview-serial

**Comment by eggfoobar** [comment]
We're currently getting hit by a known bug in the CAPI install for AWS, we'll wait on retrying serial until that is resolved.

**Comment by eggfoobar** [comment]
/lgtm

This looks good, 4.17 shows the skip happening

**Pull request merged**